daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md

```diff
@@ -149,6 +149,26 @@ To find out more details on the purpose of any of these metadata properties, ple
 
 > Note that all times are populated by the server and are not adjusted for clock skews.
 
+## Sending and receiving multiple messages
+
+Azure Service Bus natively supports sending and receiving multiple messages in a single operation. 
```

Comment:
```suggestion
Azure Service Bus supports sending and receiving multiple messages in a single operation. 
```
Reason: We should remove the term "natively" since it needs to be explained and does not add anything. Just say "Azure Service Bus supports sending and receiving multiple messages in a single operation." No native to explain, as we do not do this with other components. (e.g. We do not say "SQL Server natively supports transactions)